src/editbookmarksdialog.cpp

```diff
@@ -90,6 +92,7 @@ void EditBookmarksDialog::accept() {
 void EditBookmarksDialog::onAddItem() {
     QTreeWidgetItem* item = new QTreeWidgetItem();
     item->setData(0, Qt::DisplayRole, tr("New bookmark"));
+    item->setData(1, Qt::DisplayRole, "/");
```

Comment:
This is redundant.

Reply:
This should provide the inital value of the new bookmark item, on clicking "Add" without that it would n't it be empty until 
```
            if (!url.isValid()) {
                url = QUrl::fromUserInput(QString::fromUtf8(Fm::FilePath::homeDir().toString().get()));
            }
```
or set by the user?

Reply:
An initial value isn't needed because, with you code, an empty bookmark isn't possible on clicking OK.

BTW, this PR a good idea.

Reply:
I have updated the PR. Thanks! I have some more issues that I have found previously and am eager to fix ( I haven't worked a while on the stuff linking to fm-qt but on other parts).
I will push theses changes also in due time.

Reply:
> I have some more issues that I have found previously and am eager to fix

Looking forward to your fixes.

Please also note that you usually don't need to close a PR and make another one if something's wrong; you could fix the first one.